wls-vorfaelleundvorkommnisse-service/src/main/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/rest/ereignis/EreignisController.java

```diff
@@ -0,0 +1,55 @@
+package de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.rest.ereignis;
+
+import de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.service.EreignisService;
+import java.util.Optional;
+import lombok.val;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * This controller is responsible for mapping businessActions to REST-endpoints.
+ */
+@RestController
+@RequestMapping(value = "/businessActions")
+public class EreignisController {
+
+    private final EreignisDTOMapper ereignisDTOMapper;
+
+    EreignisService ereignisService;
+
+    public EreignisController(EreignisDTOMapper ereignisDTOMapper, EreignisService ereignisService) {
+        this.ereignisDTOMapper = ereignisDTOMapper;
+        this.ereignisService = ereignisService;
+    }
+
```

Comment:
use lombok

Reply:
see commit [f58e2b5](https://github.com/it-at-m/Wahllokalsystem/pull/412/commits/f58e2b5b367a44e8f46391ea078a62e4a1bcf65a)